README.md

```diff
@@ -41,6 +41,19 @@ Fork this repository and make changes to the forked repository. Make the changes
 #### Talks
   * [The astounding athletic power of quadcopters](https://youtu.be/w2itwFJCgFQ)
 ## Repositories
-* Autonomous landing UAV [[Link]](https://github.com/MikeS96/autonomous_landing_uav)
+
+### Repositories from BITS Pilani
+
+* GenNav ERC-BPGC [[Link]](https://github.com/ERC-BPGC/gennav)
+* Omnibase ERC-BPGC [[Link ](https://github.com/ERC-BPGC/omnibase)]
```

Comment:
One ']' missing after the 'Link' in the Omnibase link

Reply:
This is outdated. This is fixed in the latest commit